src/Juvix/Syntax/Abstract/AbstractResult.hs

```diff
@@ -14,7 +14,8 @@ import Juvix.Syntax.Concrete.Scoped.Scoper.ScoperResult
 data AbstractResult = AbstractResult
   { _resultScoper :: ScoperResult,
     _resultTable :: InfoTable,
-    _resultModules :: NonEmpty TopModule
+    _resultModules :: NonEmpty TopModule,
+    _resultExports :: HashSet NameId
```

Comment:
I would prefer to have `HashSet Name`. It seems to me that having `Name` instead of `NameId` can make tracing easier sometimes. Is there a good reason not to do that?

Reply:
The only reason is that it's more difficult because of how creating ExportInfo is implemented, and these names would have to be mapped at each stage of the pipeline (in ExportInfo the names are "S.Name' ()", so it's not even Scoped.Name yet).

I may do it if you think it's worth the effort, but it's not needed for now.

Reply:
Let's keep the `NameId`. It will be good if we can discuss later wheter having this global name info table can help us simplify things a bit.